src/sas/qtgui/MainWindow/DataExplorer.py

```diff
@@ -1132,7 +1132,7 @@ def addDataPlot2D(self, plot_set, item):
         """
         Create a new 2D plot and add it to the workspace
         """
-        plot2D = Plotter2D(self)
+        plot2D = Plotter2DWidget(parent=self, manager=self)
```

Comment:
I believe this is a result of some work that @rozyczko and @tsole0 both did independently to `DataExplorer.py` to bypass Plotter2D and call Plotter2DWidget directly to "avoid parenting mess" (closed PR #2559 ) .  I believe @tsole0 merged @rozyczko work in favor of his own. It is possible that he merged that branch into his rather than wait for it to get merged to main in order to keep his work moving? 

However, he made those changed himself originally because indeed they were required for his work so the short answer: it is in effectively in scope I believe (though this PR could have benefited from several smaller PRs )

Shorter answer .... it is no longer part of the PR changes now that main has been merged back in